driver-core/src/main/com/mongodb/operation/MapReduceToCollectionOperation.java

```diff
@@ -453,7 +476,7 @@ public void executeAsync(final AsyncWriteBinding binding, final SingleResultCall
 
     private CommandReadOperation<BsonDocument> createExplainableOperation(final ExplainVerbosity explainVerbosity) {
         return new CommandReadOperation<BsonDocument>(namespace.getDatabaseName(),
-                                                      ExplainHelper.asExplainCommand(getCommand(), explainVerbosity),
+                                                      ExplainHelper.asExplainCommand(getCommand(null), explainVerbosity),
```

Comment:
I'm not sure how best to handle situations where we don't have a description.  Only an issue for `MapReduceToCollectionOperation`